packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -1205,10 +1205,10 @@ describe("wrangler", () => {
         );
         expect(requests.count).toEqual(3);
         expect(std.out).toMatchInlineSnapshot(`
-          "Uploaded 0 of 12000.
-          Uploaded 5000 of 12000.
-          Uploaded 10000 of 12000.
-          Uploaded 12000 of 12000.
+          "Uploaded 0% (0 out of 12K)
+          Uploaded 41% (5K out of 12K)
+          Uploaded 83% (10K out of 12K)
+          Uploaded 100% (12K out of 12K)
```

Comment:
This makes me feel like we are logging the size (e.g. in Kb) of data uploaded, rather than the number of keys. Would it be clearer to have something like: `41% (5K keys out of 12K)`?

Reply:
I think 12K is much less clear than 12000 regardless of whether it has the word "keys" after it

Reply:
Yeah I admit I agree. What if we just formatted it with comma separators instead? E.g. `5,000 out of 12,000`?
@Electroid - you suggested this specific change. Can you give your view?

Reply:
I think having the full number, e.g. `12,000`, would be the most clear.

Reply:
👍